### PR TITLE
Release 6.1.2

### PR DESCRIPTION
**Pull request by quinqu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.1.2
+
+This release of teleport contains a new feature
```
Comment (r0mant) [inline CHANGELOG.md:5]: Capitalize Teleport and add a dot at the end of the sentence.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.1.2
+
+This release of teleport contains a new feature 
+
+* Added log formatting and support to enable timestamps for logs.
```
Comment (r0mant) [inline CHANGELOG.md:7]: Add a link to a pull request (see below for examples).

**Review by russjones — APPROVED** [review]
Bot.

**Review by r0mant — APPROVED** [review]

**Review by webvictim — APPROVED** [review]

**Comment by webvictim** [comment]
@quinqu Please also forward-port the CHANGELOG to `master`. Thanks!

**Pull request merged**